src/main/java/pl/byczazagroda/trackexpensesappbackend/model/FinancialTransactionCategory.java

```diff
@@ -0,0 +1,53 @@
+package pl.byczazagroda.trackexpensesappbackend.model;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import org.springframework.format.annotation.DateTimeFormat;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import javax.validation.constraints.Size;
+import java.io.Serial;
+import java.io.Serializable;
+import java.time.Instant;
+
+@Entity
+@Getter
+@Setter
+@NoArgsConstructor
+@Table(name = "financial_transaction_categories")
+public class FinancialTransactionCategory implements Serializable {
+
+    /**
+     * Class version 0.4.0.  SerialVersionUID needs to be updated with any change.
+     */
+    @Serial
+    private static final long serialVersionUID = 100040L;
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Size(max = 255)
```

Comment:
Shoud be not empty, max size 30 chars and contains only letters and digits

Reply:
Done